rust-install/src/mock/mod.rs

```diff
@@ -1,3 +1,10 @@
+//! Mocks for testing
+//!
+//! FIXM: This belongs in some other crate
+
```

Comment:
Could this module be `#![test]`? I'd just want to statically prevent multirust from calling `change_dir` which is super dangerous outside of controlled environments.